### ISSUE TITLE
operator symbols and keywords all get 'keyword.operator' scope

### ISSUE THREAD

**Issue by aeschli** [Microsoft/TypeScript-TmLanguage#37]
Try the following example in VS code (0.10.6) in the dark theme:

```typescript
module Body {
    export class Cell {
        public parent: Cell;

        constructor(parent: Cell) {
            this.parent = parent;
            if (typeof parent === 'object' || parent instanceof Error) {
            }
        }
 
```
![image](https://cloud.githubusercontent.com/assets/6461412/12139947/3fa94cca-b464-11e5-8407-46109c781345.png)

Note that `instanceof` and `typeof` are rendered white instead of blue as other keywords.

The TS grammar uses 'keyword.operator' for operator symbols like '=', '+', '!=', but also for JS keywords as switch, instanceof, import, delete, export, import, from, in, new, void, as, of.

We want to colorize the keywords, but not the symbols.

My suggestion is to use 'keyword.operator' only for symbols
 - use 'keyword' for 'instanceof', 'typeof', 'is', 'as', 'in', 'of', 'delete', 'new', 
 - for 'import' I'd suggest to use 'keyword.control.import.include'
 - for 'export' and 'void', I'd suggest 'storage.type'
 - for 'switch' use 'keyword.control'

**Comment by zhengbli** [comment]
So is it correct or not that the `module`, `typeof` and `parent` here has the `keyword.operator.ts` scope? If not, what do you suggest their scopes should be?

**Comment by aeschli** [comment]
I'd suggest to give `module`,`instanceof` and `typeof` just the `keyword.ts` scope (currently: `keyword.operator.ts`)

**Comment by aeschli** [comment]
Same problem with `namespace`

**Comment by anubmat** [comment]
Solved by pull request #105. Closing.
